test/Core.Test/Services/StripePaymentServiceTests.cs

```diff
@@ -138,6 +207,58 @@ public async void PurchaseOrganizationAsync_Stripe(SutProvider<StripePaymentServ
         ));
     }
 
+    [Theory, BitAutoData]
+    public async void PurchaseOrganizationAsync_SM_Stripe(SutProvider<StripePaymentService> sutProvider, Organization organization, string paymentToken, TaxInfo taxInfo)
```

Comment:
This test is also almost identical to `PurchaseOrganizationAsync_Stripe` so we should just modify the existing one.

Reply:
resolve